### PR TITLE
[147] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by Nurmalee**
## Task: [147] Style the Product Details Page using CSS
- Created and Styled the e-commerce Product Details Page
product-desktop-view-A
product-desktop-view-B
product-tablet-view-A
product-tablet-view-B
product-mobile-view-A
product-mobile-view-B
product-mobile-view-C

### CODE REVIEW

**Comment by Nurmalee** [comment]
@adaugochi

File: submissions/lawalNurudeen/html-forms/contact-form.html [hunk 1]
```diff
@@ -1,99 +1,137 @@
...
-                        <img src="../css/images-and-icons/social-icons/facebook logo.png" alt="">
-                        <img src="../css/images-and-icons/social-icons/twiiter logo.png" alt="">
-                        <img src="../css/images-and-icons/social-icons/gmail logo.png" alt="">
-                        <img src="../css/images-and-icons/social-icons/youtube logo.png" alt="">
-                    </div>
-                </div>
+          <div class="social-links">
+            <p>CONNECT WITH US</p>
+            <div class="social-icons">
+              <img
```
Comment (adaugochi) [inline submissions/lawalNurudeen/html-forms/contact-form.html:110]: Revert this change. All these image tags can take a single line, just as it was before

**Review by adaugochi — CHANGES_REQUESTED** [review]
Make all tag take a single line. stop breaking them, except maybe there are too long to stag on one line. and from what I am seeing, there are not

**Comment by adaugochi** [comment]
image

**Comment by adaugochi** [comment]
See that left and right marking that I did. that's where I am pointing at in my comment

File: submissions/lawalNurudeen/html-forms/contact-form.html (hunk 1 above)
Comment (Nurmalee) [reply submissions/lawalNurudeen/html-forms/contact-form.html:110]: @adaugochi those realignments were done by the PRETTIER extension I have installed on my VSCODE...

Comment (Nurmalee) [reply submissions/lawalNurudeen/html-forms/contact-form.html]: @adaugochi The changes have been made regardless

**Review by adaugochi — APPROVED** [review]
